prusti-viper/tests/hash/layout_1.rs

```diff
@@ -0,0 +1,157 @@
+use prusti_contracts::*;
+
+// Borrows are not stable since rustc::Loan indicies aren't
+// i.e. the `N` used in `DeadBorrowToken$(N)` for wands
+// will differ between compilation
+// This includes fns which return borrows, or e.g. slices
+// where the fold-unfold algorithm doesn't handle the exhale well
```

Comment:
Are the loan indices unstable because of how Polonius returns the indices, or are these renumbered in a non-deterministic fashion somewhere in Prusti? (e.g. due to the use of a randomly seeded HashSet/HashMap somewhere).

Reply:
Indeed this turned out to be caused by the use of randomly seeded `HashSet`/`HashMap` in various places. PR #817 to the `master` branch should solve this!